test/framework/catalog-events.ts

```diff
@@ -0,0 +1,17 @@
+import {
+    AdminChangedEvent,
+    OwnershipTransferredEvent
+} from '../../typechain-types/contracts/test/Box'
+import {addNamedEvent, wrapTypedEvent} from './event-wrapper'
+
+const stub = null as unknown
+const wrap = wrapTypedEvent
+
+export const Events = {
```

Comment:
question: should we have a typescript file for each set of events, or do we have all the events in this single typescript file?

Splitting them up should keep the approach scalable, as I can imagine that all the events for all the contracts would make this very large file for some projects (more of a maintenance headache).

Reply:
so I'll probably remove it completely

Reply:
Awesome 👍